### ISSUE TITLE
why multiply the angle by the lightness?

### ISSUE THREAD

**Issue by makemeunsee** [jradavenport/cubehelix#5]
https://github.com/jradavenport/cubehelix/blob/master/cubehelix.py#L99
Here you use the "fract" array to build the angles array, but "fract" contains the interpolation from minLight to maxLight, and not a simple [0..1] interpolation.
I'm not sure it's a bug but it seems weird enough to raise the question.

**Comment by jradavenport** [comment]
This is a valid point!

This goes back to the original Fortran code that D. A. Green published with his definition of the cubehelix: http://adsabs.harvard.edu/abs/2011arXiv1108.5083G

**Comment by makemeunsee** [comment]
No, it's not in the Fortran code. In the Fortran code, the `FRACT` values always span from 0 to 1, by steps of `1/(NLEV-1)`, and does not allow to specify a `minLight` nor a `maxLight` like in the port. With specified values for `minLight` and `maxLight`, the `angle` value is not making sense anymore in term of hue (0 will not be blue, nor 1 red, etc.) and the behavior will not match that detailed in the documentation of the function.
See my (untested) pull request for a possible fix.
